src/components/events/Event.jsx

```diff
@@ -0,0 +1,131 @@
+import { useParams, Link } from "react-router-dom";
```

Comment:
We no longer have events... it all got moved to activities.  we dont need this file

Reply:
I know that, dont know why I still had event lol